azkaban-common/src/main/java/azkaban/executor/MysqlNamedLock.java

```diff
@@ -0,0 +1,35 @@
+package azkaban.executor;
+
+import azkaban.db.DatabaseTransOperator;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import javax.inject.Singleton;
+import org.apache.commons.dbutils.ResultSetHandler;
+
+/**
+ *  Util class for locking using named lock in mysql
+ */
+
+@Singleton
+public class MysqlNamedLock implements ResultSetHandler<Boolean> {
+  private String getLockTemplate = "SELECT GET_LOCK('%s', %s)";
```

Comment:
Can you please make it final static string? Can you please change name to GET_LOCK_TEMPLATE? Same for release lock.

Reply:
this is a singleton class, do we need to make it static?